cocos/2d/components/label.ts

```diff
@@ -594,15 +594,150 @@ export class Label extends UIRenderer {
     @editable
     @displayOrder(18)
     @tooltip('i18n:label.underline_height')
-    public get underlineHeight (): number {
+    get underlineHeight (): number {
         return this._underlineHeight;
     }
-    public set underlineHeight (value) {
+    set underlineHeight (value) {
         if (this._underlineHeight === value) return;
         this._underlineHeight = value;
         this.markForUpdateRenderData();
     }
 
+    /**
+     ** @en
+     ** Outline effect used to change the display, only for system fonts or TTF fonts.
+     **
+     ** @zh
+     ** 描边效果组件,用于字体描边,只能用于系统字体。
+     **/
+    @editable
+    @displayOrder(19)
+    @tooltip('i18n:label.outline_used')
+    get outlineUsed (): boolean {
```

Comment:
Take a look at the names in other software

Reply:
UE： Outlined  enableShadow